lib/aws-es.js

```diff
@@ -6,75 +6,95 @@ var is = require('is_js');
 var ES_INDEX_EXISTS_STATUS_CODE = 200;
 var ES_INDEX_MISSING_STATUS_CODE = 404;
 
-function AWSES(config) {
-    if( !is.existy(config) ) throw new Error('not_config');
-    if( !is.json(config) ) throw new Error('invalid_config');
+var OPTION_TYPES = {
+    index: 'string',
+    type: 'string',
+    path: 'string',
+    accessKeyId: 'string',
+    secretAccessKey: 'string',
+    service: 'string',
+    region: 'string',
+    host: 'string',
+    id: 'string',
+    scroll: 'string',
+    scrollId: 'string',
+    body: 'json'
+};
+
+function validate_callback(callback) {
+    if (!is.existy(callback)) throw new Error('not_callback');
```

Comment:
Out of curiousity what do you get out of `is_js` that you don't get with underscore?